### PR TITLE
Append WIP to unpublished questions

### PR DESCRIPTION
**Pull request by L-Sva**
Add logic to add the string 'WIP - ' to any question that does not have the staus of 'Live on site'.

### CODE REVIEW

**Review by Aprillion — APPROVED** [review]

File: app/hooks/useQuestionStateInUrl.ts [hunk 1]
```diff
@@ -75,6 +75,11 @@ export default function useQuestionStateInUrl(minLogo: boolean, initialQuestions
       ...questionMap.get(pageid),
       questionState,
     }))
+    .map(entry => ({
+      ...entry,
+      title: entry.status === 'Live on site'
+        ? entry.title : 'WIP - ' + entry.title
```
[the remark this answers is not in the document]
Comment (plexish) [reply app/hooks/useQuestionStateInUrl.ts:81]: @L-Sva feel free to merge once you've resolved the above comment (either implement the change or give reason to prefer current).

File: app/hooks/useQuestionStateInUrl.ts [hunk 2]
```diff
@@ -75,6 +75,11 @@ export default function useQuestionStateInUrl(minLogo: boolean, initialQuestions
       ...questionMap.get(pageid),
       questionState,
     }))
+    .map(entry => ({
+      ...entry,
+      title: entry.status === 'Live on site'
```
Comment (mruwnik) [inline app/hooks/useQuestionStateInUrl.ts:80]: you can use an enum for this, i.e. 
```
import {QuestionStatus} from '~/server-utils/stampy'

entry.status === QuestionStatus.LIVE
```

**Review by Aprillion — COMMENTED** [review]
looks like we also have to load status for initial questions - I will update this PR for requested changes and also to load the status correctly

image

File: app/hooks/useQuestionStateInUrl.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Aprillion) [reply app/hooks/useQuestionStateInUrl.ts]: moved

**Review by mruwnik — APPROVED** [review]

**Pull request merged**